fcl/protodunehd/gen/prod_beam_cosmics_1GeV_protodunehd.fcl

```diff
@@ -3,7 +3,7 @@
 #include "singles_dune.fcl"
 #include "corsika_protodune.fcl"
 #include "protodunebeam.fcl"
-#include "dune_radiological_model.fcl"
+#include "dune_radiological_model_decay0.fcl"
```

Comment:
Change to dune_radiological_model_decay0_hd_common.fcl

Reply:
What is the difference between one file and the other? they look very very similar.

Reply:
What about the VD ones? should they change too?

Reply:
The second file is the updated one, although I cannot say if anything changes for the LAr radiologicals. In any case, I would reference the latest one, specially if people decide to include other HD specific background sources. For VD, if LAr is the only radiologicals so far, them it should be fine to keep this one. Otherwise there is the vd specific version of this file.